sh.ppy.osu.yaml

```diff
@@ -0,0 +1,97 @@
+app-id: sh.ppy.osu
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=all
+  - --socket=pulseaudio
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-download:ro
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=~/.wine:ro
+  - --filesystem=~/.osu:ro
+command: osu!
+modules:
+  - name: osu-lazer
+    buildsystem: simple
+    build-commands:
+        # game
+      - chmod +x ./osu.AppImage
+      - ./osu.AppImage --appimage-extract
+      - install -d -m 755 /app/bin
+      - mv ./squashfs-root/usr/bin/* /app/bin
+        # desktop file
+      - install -D -m 644 ./squashfs-root/osu!.desktop /app/share/applications/sh.ppy.osu.desktop
+      - desktop-file-edit --set-key="Icon" --set-value="sh.ppy.osu" /app/share/applications/sh.ppy.osu.desktop
+      - desktop-file-edit --set-key="Exec" --set-value="osu! %F" /app/share/applications/sh.ppy.osu.desktop
+      - desktop-file-edit --set-key="StartupWMClass" --set-value="osu!" /app/share/applications/sh.ppy.osu.desktop
+      - desktop-file-edit --set-key="StartupNotify" --set-value="true" /app/share/applications/sh.ppy.osu.desktop ###
+      - desktop-file-edit --set-key="MimeType" --set-value="application/x-osu-beatmap;application/x-osu-skin;application/x-rar;x-scheme-handler/osu" /app/share/applications/sh.ppy.osu.desktop
```

Comment:
According to https://osu.ppy.sh/wiki/en/osu%21_File_Formats, `application/x-osu-skin` doesn't exist, only `application/x-osu-skin-archive` does. Also, aren't all archive types `zip` and not `rar`?

Reply:
`zip` is correct. also it should be `x-osu-skin-archive` as you specify (also documented in https://github.com/ppy/osu/blob/master/osu.Android/OsuGameActivity.cs#L35-L37)